### PR TITLE
Add recipe for Classy Vision

### PR DESCRIPTION
**Pull request by mannatsingh**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mannatsingh** [comment]
I am willing to be listed as a maintainer

**Comment by vreis** [comment]
I am willing to be listed as a maintainer

**Comment by mannatsingh** [comment]
This PR is ready for review.

@conda-forge-admin, please ping team

File: recipes/classy_vision/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  version: {{ version }}
+
+source:
+  # NOTE: the PyPI sdist is broken for 0.1.0, use PyPI for future versions
+  url: https://github.com/facebookresearch/ClassyVision/archive/{{ version }}.tar.gz
+  sha256: e6cbb9a68a3e8ece4365f006dcb0823df85ec8b860270af5cd752f2f7ea6cbe2
+
+build:
+  number: 0
+  skip: True  # [py<36 or not linux64]
```
Comment (chrisburr) [inline recipes/classy_vision/meta.yaml]: Can this be noarch? It's okay for non linux builds to fail
```suggestion
  noarch: python
```

File: recipes/classy_vision/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
```
Comment (chrisburr) [inline recipes/classy_vision/meta.yaml]: If (and only if!) it is made noarch:
```suggestion
    - python >=3.6
```

File: recipes/classy_vision/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - pip
+  run:
+    - python
```
Comment (chrisburr) [inline recipes/classy_vision/meta.yaml]: If (and only if!) it is made noarch:
```suggestion
    - python >=3.6
```

**Comment by mannatsingh** [comment]
@chrisburr Thanks for the quick review! I hadn't used noarch python since one of the constraints mentioned in https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python wasn't satisfied ("Scripts argument in setup.py is not used" - see https://github.com/facebookresearch/ClassyVision/blob/master/setup.py#L64). But I converted to noarch and the linux build still works. I added a test to make sure the script works, and it does. 

We can continue with noarch if you think it's fine, otherwise I can go back to the previous state.

**Comment by chrisburr** [comment]
> @chrisburr Thanks for the quick review! I hadn't used noarch python since one of the constraints mentioned in https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python wasn't satisfied ("Scripts argument in setup.py is not used" - see https://github.com/facebookresearch/ClassyVision/blob/master/setup.py#L64). But I converted to noarch and the linux build still works. I added a test to make sure the script works, and it does.
> 
> We can continue with noarch if you think it's fine, otherwise I can go back to the previous state.

You're right, it can't be noarch unfortunately. If you revert 804a831 I'll merge.

**Pull request merged**